src/RoomNotifs.js

```diff
@@ -26,12 +27,12 @@ export const MUTE = 'mute';
 export const BADGE_STATES = [ALL_MESSAGES, ALL_MESSAGES_LOUD];
 export const MENTION_BADGE_STATES = [...BADGE_STATES, MENTIONS_ONLY];
 
-function _shouldShowNotifBadge(roomNotifState) {
+export function shouldShowNotifBadge(roomNotifState) {
     const showBadgeInStates = [ALL_MESSAGES, ALL_MESSAGES_LOUD];
     return showBadgeInStates.indexOf(roomNotifState) > -1;
 }
 
-function _shouldShowMentionBadge(roomNotifState) {
+export function shouldShowMentionBadge(roomNotifState) {
     return roomNotifState !== MUTE;
```

Comment:
Should this function be checking `MENTION_BADGE_STATES` instead like the other version did?

Reply:
Well, there are 4 states so one was checking if it was any of 3 of the 4, the other is checking if it is not the 4th, so they should be the same, but this is why I would like there to just be one version of this function.

Reply:
Ah, I see. From a readability perspective, it does seems a bit surprising to not use the state array defined just above in this function. It might be easier to go with the other version of the function, especially as that's a bit safer in case further states are added in the future.